quickstarts/elastic-knn-search/config.yml

```diff
@@ -0,0 +1,54 @@
+
+slug: elastic-knn-search
+description: |
+  ## Why should you monitor your usage of Elastic KNN Search?
+  Monitor your vector searches on Elastic KNN Search to get visibility on what you send to Elastic KNN Search, responses retrieved from Elastic KNN Search, latency, usage and errors. 
+
+  ### Track the query performance of your Vector DB
+  Track the behavior of your vector stores. Monitor the latency, queries, the number of documents retrieved, and the content of the documents so that you can evaluate their relevance.
```

Comment:
```suggestion
  Track the behavior of Elastic KNN Search. Monitor the latency, queries, the number of documents retrieved, and the content of the documents so that you can evaluate their relevance.
```